docs/api/platform/flow_control.md

```diff
@@ -2,51 +2,68 @@
 
 We can use Blinky to explore flow control and task management in Arm Mbed OS applications. We'll look at automated actions first, then move on to handling user actions.
 
-## Flow control for automated actions
+## Flow control for delayed actions
 
-If we want to automatically blink an LED, we have three main techniques:
+If we want to automatically blink an LED, we have four main techniques:
 
-1. [Wait](#wait)
-1. [Ticker](#ticker)
+1. [Busy wait](#busy-wait)
+1. [Ticker, Timeout](#ticker-timeout)
 1. [Thread](#thread)
+1. [EventQueue](#eventqueue)
 
-### Wait
+All above techniques accomplish delays but cater to different requirements for precision, efficiency and context.
 
-#### Busy wait
+<span class="tips">**Tip:** You may want to read the [power optimization](../apis/platform-concepts.html) tutorial to understand how to achieve power savings. </span>
+
+### Busy wait
 
-Busy wait is a method that blocks the processor for a period of time. This is an effective way to create time delays, but it is inefficient because it wastes processor time and keeps the processor running at full power for the duration of the wait:
+Busy wait is a method that blocks the processor for a period of time. The processor runs at full power for the duration of the wait:
```

Comment:
When will I want to use this?
Is it running at full power because it's working on something else? Or just because busy wait doesn't let it power down?

Reply:
Good point, now I'm discussing with the team how to better explain this.

Reply:
I've just updated the explanation